File: frontend/src/types/coupon-pack-item.ts
export interface CouponPackItemDto {
  id: number
  couponPackId: number
  couponTemplateId: number
  quantity: number
  couponTemplateName: string
}

export interface SaveCouponPackItemRequest {
  couponPackId: number
  couponTemplateId: number
  quantity: number
}

File: backend/src/FaJuan.Api/Domain/Entities/CouponPackItem.cs
namespace FaJuan.Api.Domain.Entities;

public class CouponPackItem
{
    public long Id { get; set; }
    public long CouponPackId { get; set; }
    public long CouponTemplateId { get; set; }
    public int Quantity { get; set; }
    public DateTime CreatedAt { get; set; } = DateTime.Now;
}
